components/UI/I18n/LocaleSwitch.tsx

```diff
@@ -61,12 +61,32 @@ const LocaleSwitch: React.FC<LocaleSwitchProps> = ({
         &nbsp;&nbsp;
         {languages.find((language) => language.locale === current)?.text}
       </FilterMenuButton>
-      <Menu anchorEl={anchorEl} open={open} onClose={handleClose}>
+      <Menu
+        anchorEl={anchorEl}
+        open={open}
+        onClose={handleClose}
+        sx={{ left: mobile ? "5px" : "0" }}
+        anchorOrigin={{
+          vertical: "top",
+          horizontal: "center",
+        }}
+        transformOrigin={{
+          vertical: "bottom",
+          horizontal: "center",
+        }}
+      >
         {languages.map((language) => (
           <MenuItem
             key={language.locale}
-            onClick={() => onChange(language.locale)}
+            onClick={() => {
+              onChange(language.locale);
+              handleClose();
```

Comment:
This change should be enough, what are CSS changes for?

Reply:
CSS are for better UI
Production:
![image](https://user-images.githubusercontent.com/31102191/218326884-27174b03-eb2d-4748-846f-ca10d6000c0b.png)

My development:
![image](https://user-images.githubusercontent.com/31102191/218326895-dc8c53b7-161f-4087-a7aa-43dad7af8320.png)

Reply:
@el @BerkanSezer this pr is duplicated. please visit [this pr](https://github.com/acikkaynak/deprem-yardim-frontend/pull/865/files) for merged pr

Reply:
Should i open a new pr for CSS fixes or these are unnecessary? Wdyt?